starter-code/app.js

```diff
@@ -3,3 +3,40 @@ const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
 
+app.set("views", __dirname + "/views");
+app.set("view engine", "ejs");
+
+app.get('/', (req, res) => {
+    res.render("index");
+  });
+
+app.get('/random', (req, res, next) => {
+    client
+    .getRandomJoke()
+    .then(response => {
+        res.send(`<p> ${response.value} </p>`);
```

Comment:
You want to render your view, not sending a p tag:
```
res.render('index', response.value); // Be careful, not the final code!
```
Furthermore, you send an object with render, so:
```
res.render('index', {joke: response.value});
```
Then you will acces from the view with:
```
<%= joke %>
```